scout/server/blueprints/institutes/templates/overview/clinvar_submissions.html

```diff
@@ -177,60 +179,60 @@ <h4>Variant data:</h4>
           <p>This submission is open but has no variants yet.</p>
         {% endif %}
       </div> <!--variant data div end -->
-      <div> <!--case data div -->
-        <h4>Case data:</h4>
-        {% if subm_obj.case_data and subm_obj.case_data | count_cursor > 0 %}
-        <input type="hidden" name="oldSampleName" id="oldSampleName" value="">
-        <input type="hidden" name="newSampleName" id="newSampleName" value="">
-        <table class="table table-striped">
-          <thead>
-            <tr>
-              <th></th>
-              <th>Individual ID</th>
-              <th></th>
-              <th>Case ID</th>
-              <th>Clinical features</th>
-              <th>Variant ID</th>
-              <th>Allele origin</th>
-              <th></th>
-            </tr>
-          </thead>
-          <tbody>
-            {% for case in subm_obj.case_data %}
-              <tr>
-                <td>{{loop.index}}</td>
-                <form id="rename_cd_{{subm_obj._id}}" action="{{ url_for('overview.clinvar_rename_casedata', submission=subm_obj._id, case=case.case_id, old_name=case.individual_id) }}" method="POST">
-                <td>
-                  <input type="text" name="new_name" value="{{case.individual_id}}"><button name="rename_sample" type="submit" class="btn btn-primary btn-xs mb-1"><span class="fa fa-floppy-o" aria-hidden="true"></span></button>
-                </td>
-                </form>
-                <td><button id="{{case._id}}" type="button" class="btn btn-primary btn-xs cd_btn"><span class="fa fa-search-plus" aria-hidden="true"></span></button></td>
-                <td><a href="{{ url_for('cases.case', institute_id=institute._id, case_name=subm_obj.cases[case.case_id]) }}" target="_blank">{{subm_obj.cases[case.case_id]}}</a></td>
-                <td>{{case.clin_features}}</td>
-                <td>{{var_key_name[case.linking_id]}}</td>
-                <td>{{case.allele_origin}}</td>
-                <form id="delete_casedata_{{case._id}}" action="{{ url_for('overview.clinvar_delete_object', submission=subm_obj._id, object_type='case_data') }}" method="POST">
-                  <td><button type="submit" name="delete_object" value="{{case._id}}" class="btn btn-danger btn-xs"><span class="fa fa-trash-o fa-lg" aria-hidden="true"></span></button></td>
-                </form>
-              </tr>
+      {% if subm_obj.case_data and subm_obj.case_data | count_cursor > 0 %}
+        <div id="cdata_{{subm_obj._id}}"> <!--case data div -->
```

Comment:
This document seems a lot different but the only things I did is adding an ID to this div and moving the div inside the 
`{% if subm_obj.case_data and subm_obj.case_data | count_cursor > 0 %}` condition